ui/shared/PushHealthSummary.jsx

```diff
@@ -1,33 +1,71 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import { Spinner, Table } from 'reactstrap';
-import { faHeart, faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import { Col, Row, Spinner } from 'reactstrap';
+import {
+  faHeart,
+  faCheck,
+  faExclamationTriangle,
+  faClock,
+  faChevronRight,
+} from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import broken from '../img/push-health-broken.png';
 import ok from '../img/push-health-ok.png';
 import { getPushHealthUrl } from '../helpers/url';
 
+function MetricCount(props) {
+  const { failure, inProgress } = props;
+  let color = 'success';
```

Comment:
I think you could use some of what was done here: https://github.com/mozilla/treeherder/blob/master/ui/push-health/Health.jsx#L236

Reply:
Ahh, good catch.  Maybe that function could be moved to shared and return a color as well.

Reply:
Sure, that makes sense.